### DISCUSSION TITLE
插件装上后和zotero原有的标签颜色显示功能冲突，怎么解决呢？

### DISCUSSION THREAD

**Discussion by lhjmemory** [MuiseDestiny/zotero-style#148] [Q&A] [answered]
这个插件装上后，如图中显示的文献前面的标签颜色就没了。这个有没有解决的办法，或者能不能在其他地方显示出来？
![Snipaste_2023-02-18_23-33-14](https://user-images.githubusercontent.com/33478917/219874462-aa838095-751c-45e0-9876-0b711f9c5dbe.png)

**Comment by MuiseDestiny** [comment, marked as answer]
可以的，右键标题那里，列设置，点Tags勾选。或者右键标题那里把标签列打✓

**Comment by MuiseDestiny** [comment]
如果第一次安装可以大致看看readme，对其他功能也有所了解。
